app/controllers/emails_controller.rb

```diff
@@ -1,21 +1,66 @@
 # frozen_string_literal: true
 
 class EmailsController < Sellers::BaseController
-  before_action :set_body_id_as_app
+  layout "inertia"
+
+  before_action :set_installment, only: %i[edit]
 
   def index
     authorize Installment
 
-    create_user_event("emails_view")
 
-    if request.path == emails_path
-      default_tab = Installment.alive.not_workflow_installment.scheduled.where(seller: current_seller).exists? ? "scheduled" : "published"
-      redirect_to "#{emails_path}/#{default_tab}", status: :moved_permanently
+    if current_seller.installments.alive.ready_to_publish.exists?
+      redirect_to scheduled_emails_path, status: :moved_permanently
+    else
+      redirect_to published_emails_path, status: :moved_permanently
     end
   end
 
+  def published
+    authorize Installment, :index?
+    create_user_event("emails_view")
+
+    presenter = PaginatedInstallmentsPresenter.new(seller: current_seller, type: Installment::PUBLISHED, page: 1)
+    render inertia: "Emails/Published", props: presenter.props
+  end
+
+  def scheduled
+    authorize Installment, :index?
+    create_user_event("emails_view")
+
+    presenter = PaginatedInstallmentsPresenter.new(seller: current_seller, type: Installment::SCHEDULED, page: 1)
+    render inertia: "Emails/Scheduled", props: presenter.props
+  end
+
+  def drafts
+    authorize Installment, :index?
+    create_user_event("emails_view")
+
+    presenter = PaginatedInstallmentsPresenter.new(seller: current_seller, type: Installment::DRAFT, page: 1)
+    render inertia: "Emails/Drafts", props: presenter.props
+  end
+
+  def new
+    authorize Installment
+
+    presenter = InstallmentPresenter.new(seller: current_seller)
+    render inertia: "Emails/New", props: presenter.new_page_props(copy_from: params[:copy_from])
+  end
```

Comment:
The create action should be migrated here, redirect to `#index` on success and to `#new` on failure